io.catenax.specialCharacteristics.measurement/1.0.0/Measurement.ttl

```diff
@@ -0,0 +1,63 @@
+
+
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.1.0#> .
+@prefix samm-c: <urn:samm:org.eclipse.esmf.samm:characteristic:2.1.0#> .
+@prefix samm-e: <urn:samm:org.eclipse.esmf.samm:entity:2.1.0#> .
+@prefix unit: <urn:samm:org.eclipse.esmf.samm:unit:2.1.0#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix : <urn:samm:io.catenax.specialCharacteristics.measurement:1.0.0#> .
+@prefix bpn: <urn:samm:io.catenax.shared.business_partner_number:2.0.0#> .
+@prefix samm-u: <urn:samm:org.eclipse.esmf.samm:unit:2.1.0#> .
+@prefix serialPart: <urn:samm:io.catenax.serial_part:3.0.0#> .
+
+:Measurement a samm:Aspect ;
+   samm:preferredName "SpecialCharacteristicMeasurment"@en ;
+   samm:description "Contains the Measurment for a special characteristic"@en ;
+   samm:properties ( serialPart:localIdentifiers bpn:bpnlProperty :customerPartId :characteristicId :results [ samm:property :measurementType; samm:optional true ] :revisionIndex ) ;
+   samm:operations ( ) ;
+   samm:events ( ) .
+
+:customerPartId a samm:Property ;
```

Comment:
Since the _localIdentifier_ was referenced from the SerialPart i would also suggest to use the reference _customerPartId_ from the SerialPart instead of define a new property with the name customerPartId.

Reply:
good point, will discuss this with Jochen and Sebastian. If we move the attribute into local identifier then we should explicitly reference them in the KIT

Reply:
@LuisRickert the customerPartId is **not part** of the localIdentifier, therefore **don't move this attribute into the localIdentifier** array. What i mean is to integrate the existing customerPartId attribute from the SerialPart into this aspect model. 

You've already done this with the localIdentifier:
![image](https://github.com/user-attachments/assets/058c05e2-7768-4cb0-bd2e-97c04413bcac)

### Suggestion
You should do the same with the customerPartId doing this steps:
1. **Remove** the **":customerPartId" property**
![image](https://github.com/user-attachments/assets/b196df57-60b5-42af-9e7f-6848154d6c06)

2. **Change** the defined property entry **":customerPartId"** at the Measurement aspect
![image](https://github.com/user-attachments/assets/06777744-ff24-4b4b-a39c-aa845e5e6605)

3. **to "serialPart:customerPartId"**
  ![image](https://github.com/user-attachments/assets/3af217ed-e9ac-43f5-96eb-a9778736bcbc)

4. Then the integration of the customerPartId from SerialPart is done! 😄

Reply:
ahh got it, i just had a missunderstanding :)

Reply:
done